src/uTensor/core/types.cpp

```diff
@@ -147,13 +147,13 @@ uint32_t TensorShape::num_elems() const {
 
 TensorStrides::TensorStrides(TensorShape& shape) {
   _num_dims = shape.num_dims();
-  size_t last_idx = _num_dims - 1;
+    size_t last_idx = _num_dims - 1;
```

Comment:
please remove this indent space.

Reply:
Alignment fixed. Thanks for your advice.